config/v2/component_config/standardized/OCPPCommCtrlr.json

```diff
@@ -213,12 +213,13 @@
       "variable_name": "MessageTypesDiscardForQueueing",
       "characteristics": {
         "supportsMonitoring": true,
-        "dataType": "SequenceList"
+        "dataType": "string"
```

Comment:
Unfortunate that this is necessary, there might be a slightly "hacky" workaround. According to the spec you're allowed to set a `valuesList` of a `SequenceList` to 
`* SequenceList: The (Actual) Variable value may be an
ordered (priority, etc) (sub-)set of the reported (CSV) valid
values.`
So since this variable is readonly we could just set the actual `value` and `valuesList` to the same, as long as `value` is not longer than 1000 characters...

Reply:
It is unfortunate... The `valuesList`

> specifies the allowed values for the type.

If we set it to the same value, the CSMS is not really aware that it could include other message types as well. In addition, the device model interface currently doesnt allow to change the characteristics at runtime, so I would like to keep the `string` type here...